### ISSUE TITLE
Add Apps Used Capability

### ISSUE THREAD

**Issue by r3volution11** [insidegui/Sharecuts#6]
People will want to know what apps the shortcuts use. It'll be great to include that on the front end as well as requiring people to let the app know what those are when uploading.

**Comment by banana-ben** [comment]
Depending on how active this project is, it could even parse uploaded shortcuts and detect what apps are used. It would depend on someone keeping a database of native shortcut app integrations and x-callback-urls up to date though 🤔

Ooh, maybe it’s possible to create a shortcut that exports a list of all native integrations in Shortcuts...

**Comment by r3volution11** [comment]
I agree, it wouldn't be easy by any means. But utilizing simple parsing on the description could be helpful alone to indicate apps used.

We just need to give people a reason to describe/detail what apps are used. I think upvotes and download count give people a good reason to have the best sharecut they can provide.

**Comment by gcordalis** [comment]
Adding to this if a Shortcut utilises a 3rd party API that requires an account/unique API key being able to indicate that it is a requirement would be great.

I imagine an [API] tag with the author providing a link to the service would be sufficient.

Unless there was a specific comment scheme that authors adhered to (unlikely given the wide reach Shortcuts will have) this would need to be a completely manual process by authors.
